nixos/lib/testing/macos-host.nix

```diff
@@ -0,0 +1,17 @@
+{ config, lib, hostPkgs, ... }:
+
+{
+  config = lib.mkIf hostPkgs.stdenv.hostPlatform.isDarwin {
+    defaults = {
+      virtualisation.host.pkgs = hostPkgs;
```

Comment:
This line doesn't seem darwin-specific.

Reply:
so what do you think is the best way to do it? use `!...isLinux` ?

Reply:
I think this could be set in this module, here

https://github.com/NixOS/nixpkgs/blob/589c74ae5db720612899c5b6f6e8dcdc1b97a8fc/nixos/lib/testing/nodes.nix#L29

I can't think of a situation where `host.pkgs` and `hostPkgs` should not have the same value.

Reply:
Not sure how you mean that... use `hostPkgs` instead of `...config.qemu.package`? I guess you mean that the default value for that should be hostpkgs but then in the file where this is defined?